fastlane/lib/fastlane/plugins/templates/plugin.gemspec.erb

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require '<%= require_path %>/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = '<%= gem_name %>'
+  spec.version       = Fastlane::<%= plugin_name.capitalize %>::VERSION
+  spec.author        = %q{<%= author %>}
+  spec.email         = %q{<%= email %>}
+
+  spec.summary       = %q{<%= summary %>}
+  spec.description   = %q{<%= description %>}
+  spec.homepage      = "https://github.com/<GITHUB_USERNAME>/<%= gem_name %>"
```

Comment:
https://github.com/KrauseFx/fastlane-plugin-ruby/commit/588351fdd56a6f9bac629c882ddd683f92396d61#diff-0c3ae00cec96b432904720470bc8c426R14
Could we fetch this from Git too?